packages/core/lib/patchers/http_p.js

```diff
@@ -113,7 +131,7 @@ function enableCapture(module, downstreamXRayEnabled) {
 
     var optionsCopy = Utils.objectWithoutProperties(options, ['Segment'], true);
 
-    var req = baseFunc(optionsCopy, function(res) {
+    var req = baseFunc(...(hasUrl ? [urlObj, optionsCopy] : [options]), function(res) {
```

Comment:
You must not pass the `urlObj` here, instead pass the `url` string. I just passed in `args[0]` to verify it works, but I'd suggest saving the URL string into a variable above.

Reply:
thanks! will update my PR asap!

Reply:
@willarmiros I modified the code to pass along `args[0]` unmodified in case of the new syntax (simply named it `let arg0 = args[0]` to prevent assumptions about the type, and to reuse it in several places)